pkg/apiserver/event/event.go

```diff
@@ -100,3 +103,47 @@ func (s *Service) listEvents(c *gin.Context) {
 	}
 	c.JSON(http.StatusOK, eventList)
 }
+
+// @Summary Get the events from db according to the experiment.
+// @Description Get the events from db according to the experiment.
+// @Tags events
+// @Produce json
+// @Success 200 {array} core.Event
+// @Router /api/events/listbyexperiment [get]
+// @Failure 500 {object} utils.APIError
+func (s *Service) listEventsByExperiment(c *gin.Context) {
+	startTimeStr := c.Query("startTime")
+	name := c.Query("experimentName")
+    namespace := c.Query("namespace")
+	var eventList []*core.Event
+	var err error
+	if name == "" || namespace == "" {
+		c.Status(http.StatusInternalServerError)
+		_ = c.Error(utils.ErrInternalServer.WrapWithNoMessage(fmt.Errorf("chaosName or chaosNamespace is empty")))
+		return
+	} else if startTimeStr == "" {
+		eventList, err = s.event.ListByExperiment(context.Background(), namespace, name)
+		if err != nil {
+			c.Status(http.StatusInternalServerError)
+			_ = c.Error(utils.ErrInternalServer.WrapWithNoMessage(err))
+			return
+		}
+	} else {
+		//startTime, err := time.ParseInLocation("2006-01-02 15:04:05", startTimeStr, time.Local)
+		startTime, err := time.Parse(time.RFC3339, strings.Replace(startTimeStr, " ", "+", -1))
```

Comment:
Here I have a question about how to pass time data. I didn't find a more appropriate method, do you have any opinions?

Reply:
I tried to change several time formats, but it always "not found" when searching in db.